src/azul/lambda_layer.py

```diff
@@ -23,8 +32,47 @@
 log = logging.getLogger(__name__)
 
 
+class Req(NamedTuple):
+    name: str
+    version: str
+    git: bool
+
+    wheel_re = re.compile(r'(?P<distribution>[^-]*)-'
+                          r'(?P<version>[^-]*)-'
+                          r'(?:(?P<build_tag>[^-]*)-)?'
+                          r'(?P<python_tag>[^-]*)-'
+                          r'(?P<abi_tag>[^-]*)-'
+                          r'(?P<platform_tag>[^-]*).whl')
+
+    @classmethod
+    def from_pin(cls, line: str):
+        if line.startswith('git+'):
+            line = line[len('git+'):]
+            version, name = line.split('#egg=')
+            return cls(name=name, version=version, git=True)
+        else:
+            name, version = line.split('==')
```

Comment:
Should the other operators be considered (e.g. `>=` or `-=`) https://www.python.org/dev/peps/pep-0440/#version-specifiers or is it guaranteed we'll never anything except `==`?

Reply:
We always pin all of our requirements with `==`. Other operators should raise an error.